src/modrover/learner.py

```diff
@@ -51,18 +52,26 @@ def __init__(
         param_specs: dict[str, dict],
         weights: str = "weights",
         get_score: Callable | None = None,
+        lam_ridge: float = 0.0,
     ) -> None:
         self.model_class = model_class
         self.obs = obs
         self.main_param = main_param
         self.weights = weights
         self.get_score = get_score
+        self.lam_ridge = lam_ridge
 
         # convert str to Variable
         for param_spec in param_specs.values():
-            param_spec["variables"] = list(
-                map(Variable, param_spec["variables"])
-            )
+            priors = []
+            if self.lam_ridge > 0.0:
+                priors.append(
+                    GaussianPrior(mean=0.0, sd=1.0 / np.sqrt(self.lam_ridge))
+                )
+            param_spec["variables"] = [
+                Variable(name, priors=priors)
+                for name in param_spec["variables"]
```

Comment:
What if there were other items within `param_specs`? For example, were you able to pass priors or other info before, whereas now only the name gets used?